.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug Report 🐛
+about: Something isn't working as expected? Here is the right place to report.
+---
+
+<!-- Feel free to remove sections that aren't relevant.
+
+## Title line template: [Title]: Brief description
+
+-->
+
+## Detailed description
+
+> Describe in detail the issue you're having.
+
+> Is this a feature request (new component, new icon), a bug, or a general issue?
+
+> Is this issue related to a specific component?
+
+> What did you expect to happen? What happened instead? What would you like to see changed?
+
+> What browser are you working in?
+
+> What version of the Carbon Design System are you using?
```

Comment:
@joshblack should this be carbon design system, carbon-addons-dotcom, or just remove that reference altogether?